app/views/devise/registrations/new.html.erb

```diff
@@ -29,6 +29,11 @@
             <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control" %>
           </div>
 
+          <div class="form-group">
+            <%= f.label  :consent_news_email, "Would you like to opt-in to email updates from Forward Democracy ?", class: "mb-0" %>
```

Comment:
Bearing in mind the principle "should be as punchy as possible" (e.g. as mentioned in #561):

```suggestion
            <%= f.label  :consent_news_email, "Opt in to email updates from Forward Democracy", class: "mb-0" %>
```